### DISCUSSION TITLE
Punch copy and Punch paste function icon discussion

### DISCUSSION THREAD

**Discussion by Reaper10** [tenacityteam/tenacity-legacy#371] [Ideas]
I have opened this Punch copy and Punch paste as a function and not as a plugins #370, this discussions concerns the icon designs
my mockups

 Punch copy
![punch coppy icon](https://user-images.githubusercontent.com/8137561/126881317-65a5771f-eeda-45ea-b2fc-6f60646866d6.png)

Punch paste
![punch past icon](https://user-images.githubusercontent.com/8137561/126881339-d13947c6-1b6d-4aab-bd75-63e8a6f5e2c8.png)
feed back is welcome

**Comment by poyribn** [comment]
These icons are good as placeholders.
The Default Theme is currently being changed, and the exact visual style for the redesigned icons is not being figured out yet. So I can't give any feedback on how to make the icon fit in with the others.

But what i can say, is, how you could make the icon much more clear (because it isn't very clear):
The fist looks a little bit like a fist, but not much.
I suggest looking at reference images of fists (or even making a fist with your own hand and looking at that) and then trying to replicate that hand shape.
